src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -352,7 +331,7 @@ private static Hashtable GetEditorTable(Type editorBaseType)
                 // actually run.
                 //
                 System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor(editorBaseType.TypeHandle);
-                table = s_editorTables[editorBaseType];
+                table = EditorTables[editorBaseType];
```

Comment:
@stephentoub Got it! I used a safe programming approach, but true, it is overhead. Will change it.